lint-staged.config.cjs

```
@@ -0,0 +1,12 @@
+module.exports = {
+    '*.{js,jsx,ts,tsx}': ['eslint --fix', 'prettier --write'],
+    '*.{jsx,tsx}': ['stylelint'],
+    '*.md': ['prettier --write --parser markdown'],
+    '*.json': (files) => {
+        const format = files.filter((f) => !f.endsWith('package-lock.json'));
+
+        return format.length
+            ? [`prettier --write --parser json ${format.map((f) => `"${f}"`).join(' ')}`]
+            : [];
```

ESLint

[error] Replace `⏎············?·[`prettier·--write·--parser·json·${format.map((f)·=>·`"${f}"`).join('·')}`]⏎···········` with `·?·[`prettier·--write·--parser·json·${format.map((f)·=>·`"${f}"`).join('·')}`]`

(prettier/prettier)